app/utils/apps.ts

```diff
@@ -253,3 +254,297 @@ export const makeCallErrorResponse = (errMessage: string): AppCallResponse<any>
 };
 
 export const filterEmptyOptions = (option: AppSelectOption): boolean => Boolean(option.value && !option.value.match(/^[ \t]+$/));
+
+function isAppExpand(v: unknown): v is AppExpand {
```

Comment:
I know that the mobile app has some different types than `@mattermost/types` because of casing and for compatibility, but it seems like it'd be really nice to be able to share this sort of helper function. Maybe we could have mobile share part of the types package for things like this where the casing is forced to match what the API uses

Reply:
Yes, sharing these would be great, but there are many reasons not to do so (mainly the backwards compatible nature of mobile and how it may slow the development).

But for sure something to think a bit deeper to see if we can find the correct way to do so.